node/pkg/fetcher/app.go

```diff
@@ -24,6 +29,10 @@ func New(bus *bus.MessageBus) *App {
 }
 
 func (a *App) Run(ctx context.Context) error {
+	localAggregatesChannel := make(chan LocalAggregate, LocalAggregatesChannelSize)
+	a.localAggregatesChannel = localAggregatesChannel
+	go a.bulkStoreLocalAggregates(ctx)
```

Comment:
is this also affected by `app.startAll` and `app.stopAll`?

Reply:
`app.Run()` is the function for initial execution, on app launch, while `app.startAll` and `app.stopAll` is called externally when trying to refresh the fetcher. so I think it'd be better if the `go a.bulkStoreLocalAggregates(ctx)` is included in `app.startAll` rather than `app.Run()`

Reply:
yes I think aggregation channel is also affected by `startAll` and `stopAll`. I'll move it to `startAll`